browser/src/react-redux/App/Signup/Form.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
This component looks like an input, not a form. In Signup, it looks like you are loading multiple forms. I would re-name this.

Reply:
Also, go ahead and give this component it's own folder.